src/auth/custom/custom.authenticator.ts

```diff
@@ -34,7 +34,7 @@ export class CustomAuthenticator implements IAuthenticator {
             const authHeader = request.headers['authorization'];
             const token = authHeader && authHeader.split(' ')[1];
 
-            if (token == null) {
+            if (token === 'null') {
```

Comment:
Why this type of change in check was needed? 
This is not the correct way.

Reply:
Now, I have handled both conditions.